resources/planwise/sql/scenarios.sql

```diff
@@ -46,6 +46,14 @@ UPDATE scenarios
 WHERE
   id = :id
 
+-- :name db-update-scenario-state! :! :1
+UPDATE "scenarios"
+  SET "raster" = :raster,
+      "demand-coverage" = :demand-coverage,
+      "state" = :state,
+      "updated-at" = NOW()
```

Comment:
Changing updated-at should be done `db-update-scenario!` and `db-create-scenario!`.

I actually wouldn't mind changing it in `db-update-scenario-state!` since I would expect to be an "updated by user". So we later can use it to dismiss running background jobs.